prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -0,0 +1,171 @@
+from powersimdata.utility.distance import haversine
+
+from prereise.gather.griddata.hifld.data_access import load
+
+
+def check_for_location_conflicts(substations):
```

Comment:
Does it mean in the original data, the substation list we get after the call of `get_hifld_electric_substations`, has substations with same lat/lon but different names/ids? How did people deal with it originally?

Reply:
The check comes out clean for this iteration of the HIFLD input data. I'm following the logic in the collaborators' branch, which includes such an input validation check.